README.md

```diff
@@ -1,17 +1,25 @@
-## My Project
+## amazon-kinesis-video-streams-signaling
 
-TODO: Fill this README out!
+The goal of the Signaling library is provide functionalities of control plane
+communication with AWS KVS services. Refer to [AWS KVS API Reference](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_Reference.html) for more detail.
```

Comment:
Should link to https://docs.aws.amazon.com/kinesisvideostreams-webrtc-dg/latest/devguide/kvswebrtc-websocket-apis.html as well?

Reply:
Thank you. Added to new PR.